uploadservice-ftp/src/main/java/net/gotev/uploadservice/ftp/FTPUploadRequest.kt

```diff
@@ -77,15 +77,17 @@ class FTPUploadRequest(context: Context, serverUrl: String, port: Int) :
         remotePath: String? = null,
         permissions: UnixPermissions? = null
     ): FTPUploadRequest {
-        files.add(UploadFile(filePath).apply {
-            this.remotePath = if (remotePath.isNullOrBlank()) {
-                File(filePath).name
-            } else {
-                remotePath
+        files.add(
```

Comment:
why are we modifying the upstream?

Reply:
Only the indentation is changing to accommodate lint. Otherwise the build fails on jitpack.